### ISSUE TITLE
Demo site + demo content

### ISSUE THREAD

**Issue by amdrew** [easydigitaldownloads/themedd#164]
Themedd needs a demo site. This will live over on Themedd.com. Once the demo is complete, a demo content file will be made that users can import to replicate the demo.

**Comment by pippinsplugins** [comment]
:+1:

**Comment by amdrew** [comment]
Bit of progress:

![screen shot 2017-10-18 at 19 24 09-fullpage](https://user-images.githubusercontent.com/52581/31703651-145d93f0-b43a-11e7-8b7a-72519e7d84f4.png)

**Comment by amdrew** [comment]
Created a demo logo for better visualisation

![screen shot 2017-10-18 at 22 41 27-fullpage](https://user-images.githubusercontent.com/52581/31711714-814dce9c-b455-11e7-9d68-780e5b93b62f.png)

**Comment by amdrew** [comment]
1. Added more demo products
1. Created book cover designs
1. Created book cover images for better visualisation
1. Added introduction paragraph to better explain what the site is about (in case they land on the site from Google)

![screen shot 2017-10-21 at 17 35 23-fullpage](https://user-images.githubusercontent.com/52581/31848077-52ca1606-b686-11e7-940d-301a1ee7b844.png)

___

![screen shot 2017-10-21 at 17 41 05-fullpage](https://user-images.githubusercontent.com/52581/31848103-0fe1fa1a-b687-11e7-96da-bde9641286d0.png)

**Comment by davidsherlock** [comment]
Minimal and sleek look here @amdrew!

**Comment by SDavisMedia** [comment]
LOL at those product names.

**Comment by pippinsplugins** [comment]
@amdrew any objection to having the demo site hosted on our main EDD multisite instance? something like easydigitaldownloads.com/themedd-demo

**Comment by pippinsplugins** [comment]
It could be demo.themedd.com but one of the advantages of it being on the main EDD site is the shared log ins. Also our other theme demos are there.

**Comment by SDavisMedia** [comment]
It would be nice to have it at https://easydigitaldownloads.com/themedd

**Comment by amdrew** [comment]
Demo site can be found at: https://successstories.themedd.com

Now to tackle demo content etc.

**Comment by amdrew** [comment]
Created a new issue for demo content: https://github.com/easydigitaldownloads/themedd/issues/184
